### ISSUE TITLE
Context menu fallback for custom Windows theme?

### ISSUE THREAD

**Issue by Yarrio69** [MrOtherGuy/firefox-csshacks#209]
Hi, thank you for making a CSS for [titlebar buttons fallback](https://github.com/MrOtherGuy/firefox-csshacks/blob/master/chrome/window_control_fallback_for_custom_windows_theme.css).

There's a similar case where Firefox uses high-contrast menu when custom Windows theme is set.
Can you make another style that falls back to using Proton context menu style?

Proton style vs high-contrast style:
![Untitled](https://user-images.githubusercontent.com/63135654/192166000-b225cd6d-4038-43aa-adb7-cff9c9e46819.png)

**Comment by MrOtherGuy** [comment]
I can't really test this [but there's a old gist I cooked up which is supposed to do that](https://gist.github.com/MrOtherGuy/0321315007faa7d968ae087122595964). You could test if that works.

**Comment by Yarrio69** [comment]
Nope, it's all white. This also happens when using the default Windows theme (dark).

![image](https://user-images.githubusercontent.com/63135654/192286007-25b7e266-8e42-47e5-8a6e-a5ff37aca187.png)
